From c144dc5d53ffd9a0899506a81d7104cfa03d08af Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Fri, 10 Mar 2017 14:17:59 +0800
Subject: [PATCH] video: rockchip: vcodec: fix the output buffer of jpeg

The JPEG decoder is also able to output the raw image in
NV12 format without the help of Post Processor of decoder
in VDPU1.

Change-Id: I1b29a5d93d2dd8ae2f21a7537cc3493adf57d022
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 drivers/video/rockchip/vcodec/vcodec_hw_vpu.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/vcodec/vcodec_hw_vpu.h b/drivers/video/rockchip/vcodec/vcodec_hw_vpu.h
index f2f2dac773b3..6f0a3005b45b 100644
--- a/drivers/video/rockchip/vcodec/vcodec_hw_vpu.h
+++ b/drivers/video/rockchip/vcodec/vcodec_hw_vpu.h
@@ -255,7 +255,7 @@ static struct vpu_hw_info hw_vpu_9190 = {
  * file handle translate information
  */
 DEF_FMT_TRANS_TBL(vpu_jpegd,
-		  12, 40, 66, 67
+		  12, 13, 14, 40, 66, 67
 );
 
 DEF_FMT_TRANS_TBL(vpu_h264d,
-- 
2.35.3

